sentry/src/routes/channel.rs

```diff
@@ -342,6 +336,72 @@ pub async fn get_spender_limits<A: Adapter + 'static>(
     Ok(success_response(serde_json::to_string(&res)?))
 }
 
+pub async fn get_all_spender_limits<A: Adapter + 'static>(
+    req: Request<Body>,
+    app: &Application<A>,
+) -> Result<Response<Body>, ResponseError> {
+    let channel = req
+        .extensions()
+        .get::<ChannelV5>()
+        .expect("Request should have Channel")
+        .to_owned();
+
+    let new_state = match get_corresponding_new_state(&app.pool, &channel).await? {
+        Some(new_state) => new_state,
+        None => return Err(ResponseError::NotFound),
+    };
+
+    let mut all_spender_limits: HashMap<Address, Spender> = HashMap::new();
+
+    // Using for loop to avoid async closures
+    for (spender_addr, balance) in new_state.msg.balances.spenders.iter() {
```

Comment:
What you should do is:
1. Fetch all `spendable`s
2. If there is a value in `NewState.spenders` then add `Some(SpenderLeaf)` otherwise `None`